### PR TITLE
Cypress tests

### PR DESCRIPTION
**Pull request by bogdan-zaharia**
In this PR we reorganised the file structure of cypress tests and added a test for getting jigsaw doc

### CODE REVIEW

File: cypress/FakeSVapi.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+const express = require('express');
+const app = express();
+const cors = require('cors');
+const port = 8080;
```
Comment (t-davies) [inline cypress/FakeSVapi.js:4]: FYI Cypress can intercept network requests and provide stubbed responses on it's own, if all of your requests are made client-side - so you might not need to do this.

https://docs.cypress.io/guides/guides/network-requests.html#Requests

File: cypress/integration/pages/details/activity/documents.spec.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  });
+
+  it('Requests correct JIGSAW doc', () => {
+    cy.get('.activity > table > tbody > tr > td > p > strong > button')
+      .eq(-2)
+      .should('contain', 'Document');
+
+    cy.get(
+      '#main-content > div > div:nth-child(2) > div.details__right-column > div.activity > table > tbody > tr:nth-child(72) > td:nth-child(3) > p:nth-child(2)'
+    ).should('contain', 'JIGSAW');
```
Comment (t-davies) [inline cypress/integration/pages/details/activity/documents.spec.js:49]: These chained selectors will make for quite brittle tests, as if anything in the structure of the page changes the tests will break. You could add `data-testid` attributes to the things you're interested in inspecting to make finding them easier - see https://kentcdodds.com/blog/making-your-ui-tests-resilient-to-change/

File: cypress/fixtures/customerNotes.json [hunk 3]
```diff
@@ -0,0 +1,299 @@
...
...UNION, ACCOUNT HOLDER:ROGER BROWN, AMOUNT:0.2, ACCOUNT NUMBER:10160892, BRANCH:608367]. THE CLAIMANT GIVES PERMISSION TO SHARE INFORMATION WITH THEIR LANDLORD OR REPRESENTATIVE ABOUT THE STATUS OF THEIR HOUSING BENEFIT CLAIM. THE CLAIMANT STATED THEY HAVE 2 BEDROOMS. DEPENDANT CHILD JEVON BROWN HAS BEEN PLACED UNDER THE CLAIMANT/PARTNER'S GUARDIANSHIP. DEPENDANT CHILD KADEHDRA JOHNSON HAS BEEN PLACED UNDER THE CLAIMANT/PARTNER'S GUARDIANSHIP.Household Type = HOMELESSSHORTTERMLEASEDRelationshipWithOtherNonDependant - InRelationship = noUnearnedIncome-Description = CHILD BENEFITUnearnedIncome-Description = CHILD TAX CREDITUnearnedIncome-Description = WORKING TAX CREDITHouseholdMember-RelationshipToClaimant = sonRelationshipWithOtherNonDependant - InRelationship = noHouseholdMember-RelationshipToClaimant = step-daughterRelationshipWithOtherNonDependant - InRelationship = noRent Free Weeks = 0THE CLAIMANT WOULD LIKE TO BACKDATE AND THEIR CIRCUMSTANCES WERE THE SAME AS THEY ARE NOW. THE CLAIMANT STATED THE FOLLOWING REASON FOR BACKDATING [ THE REASON WHY I HAVE NOT BEEN ABLE TO SUBMIT THE HOUSING BENEFIT FORM I'VE BEEN UNABLE TO GET ACCESS TO MY PREVIOUS PREMISES TO GET MY DOCUMENTS ]",
```
Comment (t-davies) [inline cypress/fixtures/customerNotes.json:137]: I'm sure they are, but just double checking these are fake details?

File: cypress/FakeSVapi.js (hunk 1 above)
Comment (kosiakkatrina) [reply cypress/FakeSVapi.js:4]: I might be wrong, but I think they only intercept XHR and not fetch requests T_T

Comment (t-davies) [reply cypress/FakeSVapi.js:4]: Ahh yeah, good spot! Thanks Kat - you're right, `fetch` isn't not supported by the looks of it - weird, I would have expected them to support that.

https://github.com/cypress-io/cypress/issues/95

**Pull request closed without merging**